ontology.json

```
@@ -0,0 +1,63 @@
+  "home": "🏠",
+  "files": "📁",
+  "up": "⬆️",
+  "source": "💡",
+  "termux": "📱",
+  "pick": "⛏️",
+  "git": "🌳",
+  "sublime": "📝",
+  "rust": "🦀",
+  "minizinc": "🧩",
+  "rs": "💻",
+  "syntaxes": "✍️",
```

Biome (2.1.2)

[error] The key minizinc was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)